src/bundle/Resources/public/scss/_mixins.scss

```diff
@@ -1,12 +1,14 @@
 @import 'mixins/image-preview';
+@import 'mixins/type-icon';
+@import 'mixins/inputs';
 
 @mixin checkbox-icon() {
     border: calculateRem(1px) solid $ez-color-secondary;
     margin: 0;
     padding: 0;
     width: calculateRem(64px);
     height: calculateRem(32px);
-    border-radius: calculateRem(2px);
+    border-radius: $border-radius;
```

Comment:
Let's not use bootstrap variables in our SCSS. Please create `ibexa` prefixed variable.